Grievance_Backend/grievance/src/main/java/com/grievance/exception/DepartmentAlreadyExists.java

```diff
@@ -10,16 +10,17 @@
  */
 @ResponseStatus(
   value = HttpStatus.CONFLICT,
-  reason = "Department with this email already exist"
+  reason = "Department with this Title already exists"
```

Comment:
can create a Constant class for Exception message and use that constant like above HttpStatus.CONFLICT.

Reply:
we don't need to create two different Exception class for DepartmentAlreadyExists and EmployeeAlreadyExistsException, we can create single Exception RecordAlreadyExistsException and change the message of exception accordingly.

Reply:
add global handler to handle the exception.

Reply:
Creation of constants are done.
Generic Record already exists classes is created and redundant classes are removed.
Global exception handler is completed.